app/views/research_sessions/preview.html.erb

```diff
@@ -198,7 +198,7 @@
       to take part it will not affect the service <%= you_or_they %> receive.
     </p>
     <p>
-      <%= you_or_they.capitalize %> can change your mind at any point for any
+      <%= you_or_your_child.capitalize %> can change <%= your_or_their %> mind at any point for any
```

Comment:
The content here is not reflecting the commit message. It still reads:

> "mind at any point for any reason - just let us know!"
instead of 
> "during the session at any point for any reason by letting the researcher know **you / they**
want to stop."

Reply:
Good spot - rebased